src/pepr/operator/controllers/network/generators/ambientHealthprobes.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright 2024 Defense Unicorns
+ * SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+ */
+
+import { Direction } from "../../../crd";
+import { generate } from "../generate";
+
+// See https://istio.io/latest/docs/ambient/usage/networkpolicy/#ambient-health-probes-and-kubernetes-networkpolicy
+export const allowAmbientHealthprobes = (namespace: string) =>
+  generate(namespace, {
+    direction: Direction.Ingress,
+    description: "Ambient Healthprobes",
+    remoteCidr: "169.254.7.127/32",
```

Comment:
Are we sure this hardcoded cidr will be stable over time?

Reply:
I'm not sure this is a suitable answer but this IP comes [Istio's documentation](https://istio.io/latest/docs/ambient/usage/networkpolicy/#ambient-health-probes-and-kubernetes-networkpolicy) where they state:
> by default ambient uses the link-local address 169.254.7.127 to identify and correctly allow kubelet health probe packets

From what I can tell this is an IP that Istio has chosen so I think we can expect it to be stable unless they make a breaking change?

Reply:
That makes sense. We should see a failure in the tests. 

Would you mind to add a small comment here with the link to the Istio doc for future maintainers?

Reply:
Docs link is above on line 9 🙂